docs/core/install/templates.md

```diff
@@ -0,0 +1,233 @@
+
+```console
+dotnet new -u
+Template Instantiation Commands for .NET Core CLI
+
+Currently installed items:
+
+... cut to save space ...
+
+  c:\code\nuget-packages\some-folder
+    Templates:
+      A Template Console Class (templateconsole) C#
+      Project for some technology (contosoproject) C#
+    Uninstall Command:
+      dotnet new -u c:\code\nuget-packages\some-folder
+```
+
+## Install templates from other SDKs
+
+If you've installed each version of the SDK sequentially, for example you installed SDK 2.0, then SDK 2.1, and so on, you'll have every SDK's templates installed. However, if you start with a later SDK version, like 3.1, only the templates for [LTS (long term support) releases](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) are included, which at the time of the SDK 3.1 release is SDK 2.1 and SDK 3.1. Templates for any other release aren't included.
+
+The .NET Core templates are available on NuGet, and you can install them like any other template. For more information, see [Install NuGet hosted package](#nuget-hosted-package).
+
+| SDK              | NuGet Package Identifier                                                                                                      |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| .NET Core 2.1    | [`Microsoft.DotNet.Common.ProjectTemplates.2.1`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.2.1) |
+| .NET Core 2.2    | [`Microsoft.DotNet.Common.ProjectTemplates.2.2`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.2.2) |
+| .NET Core 3.0    | [`Microsoft.DotNet.Common.ProjectTemplates.3.0`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.3.0) |
+| .NET Core 3.1    | [`Microsoft.DotNet.Common.ProjectTemplates.3.1`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.3.1) |
+| ASP.NET Core 2.1 | [`Microsoft.DotNet.Web.ProjectTemplates.2.1`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.2.1)       |
+| ASP.NET Core 2.2 | [`Microsoft.DotNet.Web.ProjectTemplates.2.2`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.2.2)       |
+| ASP.NET Core 3.0 | [`Microsoft.DotNet.Web.ProjectTemplates.3.0`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.3.0)       |
+| ASP.NET Core 3.1 | [`Microsoft.DotNet.Web.ProjectTemplates.3.1`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.3.1)       |
+
+For example, the .NET Core SDK includes templates for a console app targeting .NET Core 2.1 and .NET Core 3.1. If you wanted to target .NET Core 3.0, you would need to install the 3.0 templates.
+
+01. Try creating an app that targets .NET Core 3.0.
```

Comment:
nit: The leading 0 isn't necessary.

Reply:
I do that so that tabbing is even at 4-spaces in. 😁